client/my-sites/plugins/plugin-details.jsx

```diff
@@ -391,10 +401,25 @@ function PluginDetails( props ) {
 					</div>
 
 					<div className="plugin-details__actions">
-						<PluginDetailsCTA plugin={ fullPlugin } isPlaceholder={ showPlaceholder } />
-
-						{ ! showPlaceholder && ! requestingPluginsForSites && (
-							<PluginDetailsSidebar plugin={ fullPlugin } />
+						<div className="plugin-details__sidebar">
+							<PluginDetailsCTA plugin={ fullPlugin } isPlaceholder={ showPlaceholder } />
+
+							{ ! showPlaceholder && ! requestingPluginsForSites && (
+								<PluginDetailsSidebar plugin={ fullPlugin } />
+							) }
+						</div>
+
+						{ ! showPlaceholder && ! requestingPluginsForSites && isWporgPluginFetched && (
+							<Card className="plugin-details-download-card">
+								<Gridicon icon="cloud-download" size={ 48 } />
+								<p>{ downloadText }</p>
+								<Button
+									href={ `https://downloads.wordpress.org/plugin/${ fullPlugin?.slug || '' }.zip` }
```

Comment:
@WBerredo If/when you're doing a second pass at this, maybe `fullPlugin?.slug` could become part of the conditionals to show this text, as [a download link without the slug](https://downloads.wordpress.org/plugin/.zip) just leads to a 404

Reply:
This shouldn't happen as we check for `isWporgPluginFetched` and when we have a successful fetch this request brings the slug. 

The fallback to `|| ''` was more in the intention to be super safe and not break anything.

That said, do you still feel we need this check on the top?

Reply:
Probably not, although it sounds like you actually want to know if the slug is available, not whether the plugin is fetched? Either way, sounds like it's a non-issue.

Reply:
> sounds like you actually want to know if the slug is available, not whether the plugin is fetched

There are two sources for plugins: DotCom and DotOrg. Both of them will provide a plugin slug. 
In this case, the verification is if the source is DotOrg, because only in a positive case this link will work for downloading the zip file with the code.